packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -338,14 +282,16 @@ export default connect(
     profile: profile(state),
     plans: plans(state),
     createSubscriptionStatus: createSubscriptionStatus(state),
+    updateSubscriptionPlanStatus: updateSubscriptionPlanStatus(state),
     plansByProductId: plansByProductId(state),
   }),
   {
-    resetCreateSubscription: resetCreateSubscription,
```

Comment:
Also a weird thing: `resetCreateSubscription` and `resetCreateSubscriptionError` are both the same thing. And they don't really get used differently anywhere. So, I just collapsed `resetCreateSubscriptionError` into `resetCreateSubscription`